webapp/src/components/react_select_setting.jsx

```diff
@@ -34,14 +36,14 @@ export default class ReactSelectSetting extends React.PureComponent {
     }
 
     componentDidMount() {
-        if (this.props.addValidate && this.props.name) {
+        if (this.props.addValidate) {
```

Comment:
Why was the `this.props.name` check removed?

Reply:
Currently, the validator requires a name to register a form field to validate. When we have dynamically created fields (like several rows of filters), there is not a unique name to use per form field (even the name of the selected Jira field won't work because our design supports multiple rows of the same Jira field). So I'm using the reference of the `isValid` instance method as the unique identifier when the name is not available.

afaik we actually don't receive any benefit from using a name for registering the required form field (like some sort of debugging), so if we choose to go through with the method I have in the code here, I'm thinking we also get rid of the name portion of the validation registration for consistency.

Reply:
Sounds good to me! (removing for consistency)